src/providers/ldap/sdap_async_nested_groups.c

```diff
@@ -1633,6 +1633,8 @@ sdap_nested_group_single_step_process(struct tevent_req *subreq)
         } else {
             DEBUG(SSSDBG_OP_FAILURE, "Unknown entry type [%s]!\n",
                   state->current_member->dn);
+            DEBUG(SSSDBG_TRACE_FUNC, "Consider enabling sssd-ldap option "
+                                     "ldap_ignore_unreadable_references\n");
```

Comment:
Hi,

is there a reason you used `SSSDBG_TRACE_FUNC`? The `Unknown entry type` will be printed on the default level but your hint will be only shown at level 6 or higher?

bye,
Sumit

Reply:
Thanks